src/elements/stories/gradientTextInput.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import { Stack } from "@mui/material";
+import GradientTextInput from "../GradientTextInput";
+
+export default {
+  title: "Gradient Text Input",
+  component: GradientTextInput,
+};
+
+export const Variations = () => (
+  <Stack mt={ 2 } direction="column" spacing={ 4 } display="flex">
+    <GradientTextInput textAlign="center" autoFocus />
+    <GradientTextInput textAlign="center" errorMessage="With error message" />
```

Comment:
Would it be good to make a story with this component inside a Formik wrapper with some Yup validation? For example, if the user inputs any invalid characters like @ it will show the correct error (as shown [here](https://www.figma.com/file/RS5VWuUtH3f5NRHJWdhcZK/NEWM?node-id=935%3A7974)) I think this could be another story inside the form storybook folder. Up to you tho

Reply:
I think that would be good to do. I created the input components before I implemented story book, so I had just made them static. I think making the stories dynamic using Formik would be good.